ui/src/components/query/viewer/RecursiveMatchDisplay.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <component
+      :is="!inline ? 'div' : 'span'"
+      :style="indentationStyle(inline,depth)">
+    <span v-if="match.includeIf" :class="'then'">{{ match.includeIf }}</span>
+    <span v-if="match.inlineOperator" :class="match.inlineOperator">{{ match.inlineOperator}}</span>
+    <span v-if="match.exclude" :class="'field'">NOT</span>
+    <span v-if="match.orderBy" :class="'field'"v-html="match.orderBy.description"></span>
```

Comment:
incorrect use of v-html on standard string
v-html is a potential security issue and is used to render secure html strings only.
Use `<pre></pre>` element if you need string formating